spot_driver/spot_driver/spot_ros2.py

```diff
@@ -72,6 +72,11 @@
 MAX_DURATION = 1e6
 MOCK_HOSTNAME = "Mock_spot"
 
+COLOR_END    = '\33[0m'
+COLOR_RED    = '\33[31m'
+COLOR_GREEN  = '\33[32m'
+COLOR_YELLOW = '\33[33m'
```

Comment:
Instead of this use `self.node.get_logger().info/warn/error`.  They should show up colored - sometimes docker does dumb stuff with `tty` but we should fix that inside docker.  Anything that goes into the logger will also get put into the rosbag so that's important.